src/parser/on_demand/parse_to_elements.ts

```diff
@@ -9,29 +9,31 @@ import { NumberUtils } from '../../utils/number_utils';
  * - `minKey` is set to 255 so unsigned comparisons succeed
  * - Modify with caution, double check the bundle contains literals
  */
-const enum BSONElementType {
-  double = 1,
-  string = 2,
-  object = 3,
-  array = 4,
-  binData = 5,
-  undefined = 6,
-  objectId = 7,
-  bool = 8,
-  date = 9,
-  null = 10,
-  regex = 11,
-  dbPointer = 12,
-  javascript = 13,
-  symbol = 14,
-  javascriptWithScope = 15,
-  int = 16,
-  timestamp = 17,
-  long = 18,
-  decimal = 19,
-  minKey = 255,
-  maxKey = 127
-}
+const BSONElementType = {
```

Comment:
erasableSyntaxOnly does not permit enums.

Reply:
This may not be a desirable change

This:
<img width="422" alt="Screenshot 2025-04-30 at 10 54 40 AM" src="https://github.com/user-attachments/assets/714bba63-94e9-422e-a63c-a88de8c66348" />

Becomes:
<img width="422" alt="Screenshot 2025-04-30 at 10 53 48 AM" src="https://github.com/user-attachments/assets/f13dab2d-93a8-4844-9752-c51fd3ac82dd" />

I offer #794 as an optional alternative. I think it's nice to keep the literals